content/300-guides/200-deployment/110-deployment-guides/450-deploying-to-cloudflare-workers.mdx

```diff
@@ -73,6 +73,19 @@ Wrangler is the official Cloudflare Worker CLI. You will use it to develop and d
 
 ```terminal
 npx wrangler init
+ ⛅️ wrangler 2.1.15
```

Comment:
This looks to be the output for the command. Need to remove this and just explain the input steps, or maybe use the `<CodeWithResult>` component if it's important